src/cli.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+import {execSync} from 'child_process';
+import * as path from 'path';
+import * as shim from './shim';
+
+// Parameters can be provided in the forms of:
+// $ rss googleapis
+// $ rss googleapis@20
+// $ rss googleapis@28.1.0
+// $ rss googleapis output.trace
+
+const args = process.argv.slice(2);
+if (args.length < 1) {
+  console.error(`USAGE: rss some-npm-module [output]`);
```

Comment:
That comment up there ^^^^, yes that one. It would be so nice to show that as part of the usage help.